floyd/client/data.py

```diff
@@ -61,6 +62,7 @@ def new_tus_credentials(self, data_id):
 
     def get(self, id):
         try:
+            id = normalize_data_name(id)
```

Comment:
I'm not very sure if the normalization should happen here in the client. I would have expected this to happen in the cli code similar to your change in `cli/run.py`

Reply:
You are right. That is a better approach. This was a bit of a shortcut :flushed: I'll clean things up